### PR TITLE
added: cartpage

### PR DESCRIPTION
**Pull request by sharmayatendra**
# Cart Page Added

## Design of cart page with two cards one as an item & the other for a placed order with order details.

image

### CODE REVIEW

File: Pages/Cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+.price-value {
+    color: var(--primary-color);
+    margin: 0;
+}
+
+.card-btn {
+    margin-bottom: 4px;
+    margin-right: 4px;
```
Comment (yashpatel1797) [inline Pages/Cart/cart.css]: use shorthand properties.

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

File: Pages/Cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+.quantity {
+    display: flex;
+    align-items: center;
+}
+
+.quantity-heading {
+    margin-right: 1rem;
+    margin-left: 0;
```
Comment (bishnoimukesh) [inline Pages/Cart/cart.css]: use shorthand

File: Pages/Cart/cart.css [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+.card-btn {
+    margin-bottom: 4px;
+    margin-right: 4px;
+    font-size: 1.1em;
+}
+
+.plus-icon , .minus-icon {
+    cursor: pointer;
+    margin: 0 0.5rem 0 0.5rem;
```
Comment (bishnoimukesh) [inline Pages/Cart/cart.css]: use shorthand

**Review by bishnoimukesh — COMMENTED** [review]
suggestion given rest looks good to merge

File: Pages/Cart/cart.css (hunk 1 above)
Comment (sharmayatendra) [reply Pages/Cart/cart.css]: done

File: Pages/Cart/cart.css (hunk 2 above)
Comment (sharmayatendra) [reply Pages/Cart/cart.css]: done

**Comment by HarshnaNHaswani** [comment]
Hi, I noticed that the buttons are not properly center-aligned, add proper spacing between them, one of them can be a secondary button. Both look like primary now
image
Also, you can highlight the total amount in your cart

**Pull request merged**